### PR TITLE
Add R package pgirmess.

### PR DESCRIPTION
**Pull request by jdblischak**
CRAN R package [pgirmess](https://cran.r-project.org/package=pgirmess). Recipe created with conda-build 2.1.18 and [conda_r_skeleton_helper](https://github.com/bgruening/conda_r_skeleton_helper).

### CODE REVIEW

**Comment by jdblischak** [comment]
The Travis build failed because it pulled [r-maptools](https://github.com/conda-forge/r-maptools-feedstock) from the defaults channel:

https://travis-ci.org/conda-forge/staged-recipes/builds/339616600#L285

And r-maptools failed during feedstock creation because it pulled r-foreign and r-sp from defaults:

https://travis-ci.org/conda-forge/r-maptools-feedstock/jobs/266322503#L237

File: recipes/r-pgirmess/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - r-spdep >=0.5_43
+    - r-splancs >=2.01_31
+
+test:
+  commands:
+    - $R -e "library('pgirmess')"  # [not win]
+    - "\"%R%\" -e \"library('pgirmess')\""  # [win]
+
+about:
+  home: http
```
Comment (cbrueffer) [inline recipes/r-pgirmess/meta.yaml]: The homepage wasn't added correctly by the skeleton helper.

Comment (jdblischak) [reply recipes/r-pgirmess/meta.yaml]: Good catch, @cbrueffer! I fixed that [bug](https://github.com/conda/conda-build/blob/master/CHANGELOG.txt#L761) back in August, but unfortunately the fix went into conda-build 3 and thus continues to cause problems with conda-forge recipes.

**Comment by jdblischak** [comment]
I had restarted the [Travis build for r-maptools](https://travis-ci.org/conda-forge/r-maptools-feedstock), which resulted in the package for R 3.4.1 on osx-64 to be built and uploaded to [Anaconda Cloud](https://anaconda.org/conda-forge/r-maptools/files?version=0.9_2). And now it builds on macOS!

**Comment by jdblischak** [comment]
@bgruening Please merge

**Comment by bgruening** [comment]
Of course! :)

**Pull request merged**